libs/community/langchain_community/callbacks/__init__.py

```diff
@@ -72,6 +72,10 @@
     from langchain_community.callbacks.trubrics_callback import (
         TrubricsCallbackHandler,
```

Comment:
**Static Code Issue**: Move import 'langchain_community.callbacks.trubrics_callback.TrubricsCallbackHandler' out of type-checking block. Import is used for more than type hinting. <br> **Fix**: Move the import statement outside the TYPE_CHECKING block to the top-level imports. <br> **Code Suggestion**: 
 ```
 -    from langchain_community.callbacks.trubrics_callback import (
 -        TrubricsCallbackHandler,
 -    )
 +from langchain_community.callbacks.trubrics_callback import TrubricsCallbackHandler
 ```